tests/helpers/linux/test_tc.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) ByteDance, Inc. and its affiliates.
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+from random import choice
+from string import ascii_letters
```

Comment:
I'd prefer just importing the top-level modules here so it's clearer in the code what the imported functions are.

```suggestion
import random
import string
```